### DISCUSSION TITLE
Skywalking Kafka Client TLS enabled

### DISCUSSION THREAD

**Discussion by skundu1234** [apache/skywalking#8672] [General]
We need to get the details where we can keep cert paths to enable TLS for Kafka Client and we did not get much details from below link.
We need to understand where we can keep those TLS configuration(which .yaml file).Please help on this.

https://skywalking.apache.org/docs/skywalking-satellite/latest/en/setup/plugins/client_kafka-client/

**Comment by wu-sheng** [comment]
Have you read the whole doc? Especially about.
I think there is only one config file, `satellite_config.yaml`.

**Reply by wu-sheng** [reply]
kafka client plugin should be added by yourself. We provided an example too, https://skywalking.apache.org/docs/skywalking-satellite/latest/en/setup/examples/feature/transmit-log-to-kafka/readme/

**Comment by skundu1234** [comment]
Thanks for that help.
We have enabled kafka plugin from agent.config and also gave configuration in satellite_config.yaml. Should we use both to connect Kafka . 
If satellite_config.yml is the way to connect to Kafka, where we should add topic related details.

**Reply by wu-sheng** [reply]
I don't know what do you plan to do. Should or should not is your decision.

**Reply by skundu1234** [reply]
We want to connect kafka through TLS. Referring your solution, we used satellite_config.yml for this.
But where to add kafka topic related details in that configuration.

**Reply by wu-sheng** [reply]
I don't think you need to config topics manually.

**Reply by wu-sheng** [reply]
If you are using Kafka, `agent->Kafka->OAP` server should be enough.

**Reply by skundu1234** [reply]
We tried this but how to add TLS related config in agent.config to connect kafka.
